perception/multi_object_tracker/src/multi_object_tracker_core.cpp

```diff
@@ -225,24 +237,32 @@ std::shared_ptr<Tracker> MultiObjectTracker::createNewTracker(
 void MultiObjectTracker::onTimer()
 {
   const rclcpp::Time current_time = this->now();
-  const auto self_transform =
-    getTransformAnonymous(tf_buffer_, world_frame_id_, "base_link", current_time);
-  if (!self_transform) {
-    return;
+  // check the publish period
+  const auto elapsed_time = (current_time - last_published_time_).seconds();
+  // if the elapsed time is over the period, publish objects with prediction
+  constexpr double maximum_latency_ratio = 1.11;  // 11% margin
```

Comment:
[imo]
Since early publish is better for the planning module, I prefer larger value like 1.4 or more.

Reply:
the acceptable publish interval is set to 70% to 111%, as agreed with @yukkysaito